consensus/misc/curie.go

```diff
@@ -0,0 +1,14 @@
+package misc
+
+import (
+	"github.com/scroll-tech/go-ethereum/core/state"
+	"github.com/scroll-tech/go-ethereum/log"
+	"github.com/scroll-tech/go-ethereum/rollup/rcfg"
+)
+
+// ApplyCurieHardFork modifies the state database according to the Curie hard-fork
+// rules, updating the bytecode of the L1GasPriceOracle contract.
+func ApplyCurieHardFork(statedb *state.StateDB) {
+	log.Info("Applying Curie hard fork")
+	statedb.SetCode(rcfg.L1GasPriceOracleAddress, rcfg.CurieL1GasPriceOracleBytecode)
```

Comment:
we should also set storage slots for Curie here. Otherwise, the these variables will have zero value right after fork unless owner send some transactions.

`l1BlobBaseFee`, `commitScalar`, `blobScalar` and `isCurie`.

Reply:
Makes sense. But what should be the initial values?

Reply:
For `l1BlobBaseFee` can be set to `1`, since in mainnet it is almost `1` in all the time.

For `commitScalar` and `blobScalar`, we can follow the value in [this doc](https://www.notion.so/scrollzkp/Fees-after-Curie-672a24ad77e847d8aeee4661fcec49e8). Currently, it says `commitScalar=230759955285` and `blobScalar=417565260`.

For `isCurie`, simply set it to `true` is enough.

Reply:
Done in [b8581d5](https://github.com/scroll-tech/go-ethereum/pull/755/commits/b8581d5f145bc875e8fd9ef12b1da6913af210ea).